plugins/foreman_remote_execution/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Remote Execution Documentation
-versions: [0.1, 0.0]
+versions: [nightly, 0.1, 0.0]
```

Comment:
ok, I was thinking that people might have git checkouts but let's remove it for now so it's less confusing and we'll see if it would be useful in future